src/selectfield/selectfield.scss

```diff
@@ -148,3 +163,20 @@ $selectfield-error-color: unquote("rgb(222, 50, 38)") !default;
     visibility: visible;
   }
 }
+
+// SelectField Icon
+.mdl-selectfield__icon {
+  bottom: $selectfield-vertical-spacing;
+  cursor: pointer;
+  color: $selectfield-label-color;
+  left: 0;
+  padding: 2px 0;
```

Comment:
ah. the top padding. didn't see that at first. Could go back in and leverage a variable.